G7SensorKitUI/Views/G7SettingsView.swift

```diff
@@ -39,6 +39,7 @@ struct G7SettingsView: View {
 
         formatter.dateStyle = .short
         formatter.timeStyle = .short
+        formatter.dateFormat = "E, MMM d, h:mm a"
```

Comment:
We prefer not to specify date formats, so we can benefit from built in iOS locale support.  Some locales do not prefer am/pm.

Reply:
Does the use of the following satisfy this?:
   formatter.locale = Locale.current
   formatter.setLocalizedDateFormatFromTemplate("E, MMM d, hh:mm")

Reply:
Does this work on small screens? Can you please check on iPhone SE?

Reply:
I don't have an actual SE device to test this on. Simulator doesn't connect to real G7. Open to suggestions or help on how to test on SE.